website/docs/foundations/typography/partials/code/how-to-use.md

```diff
@@ -1,64 +1,58 @@
-The suggested way to apply the typographic definitions to a UI element is using the **predefined CSS helper classes** provided.
+## How to use these styles
 
-#### Design tokens
+While we do offer design tokens for typography, we recommend using the predefined CSS helper classes.
 
-There are numerous [typographic design tokens](./tokens) in our system.
+### CSS helper classes
 
-Since they are "atomic" definitions that associate a particular typographic property (e.g. `font-size`, or `line-height`, or `letter-spacing`) to a single value, they are not exactly "typographic" styles.
+In a single declaration, these classes contain everything to apply a standard style to an element: `font-family`, `font-size`, `line-height`, and a reset for `margin` and `padding` to match the intended design language.
```

Comment:
Would you want to clarify that this applies a "standard **_typographic_** style"?